releases/development/master/extra/barclamp_install.rb

```diff
@@ -35,6 +35,17 @@ def usage()
   exit
 end
 
+def naviational_barclamps
+  [
+    "cisco-ucs"
+    "crowbar",
+    "network",
+    "openstack",
+    "swift",
+    "tempest",
```

Comment:
Navigation was inside the nav key before too. It's just what is under nav that changed.